### DISCUSSION TITLE
More data from the Nordic region

### DISCUSSION THREAD

**Discussion by ValterFallenius** [openclimatefix/metnet#27] [Show and tell]
I have a lot of data I can share with you. We spoke about adding my dataset to Kaggle a few months ago, now I have gotten an approval from my supervisor. 

@jacobbieker are you still interested in this? 

I have used a data field corrected from ground observations. This one I have packed 500k snapshots in an .h5-file of from Swedish radar stations, every 5 minutes for ~4 years 2017-2021. Is this good enough?

**Comment by JackKelly** [comment]
(just to flag up that Jacob is on holiday for a week or so!)

**Comment by jacobbieker** [comment]
Hi, yeah, I'd be very interested! Like Jack said I'm on vacation till the 5th of April, but then I'd love to help make that data be public.

**Reply by ValterFallenius** [reply]
Sounds good.

**Reply by jacobbieker** [reply]
Hi, I'm back from vacation now, so would love to start on making thisdata public! Is there a place where I could get the data?

**Reply by ValterFallenius** [reply]
I will get you a link asap. Cluster is down for maintenance today.

**Reply by ValterFallenius** [reply]
Do you want the dataset with all the fields, see plot here:

![second_plot](https://user-images.githubusercontent.com/21970939/162155340-2879170b-2106-40d9-b1d9-119d9eb372d3.png)

Or do you want the radar composites data3 (the one I'm using) which is corrected against ground observations.

If you want all data fields it is rather big, 530.000 files and it comes in individual h5 files. Since I have used data3 I have it packaged into a single 14 GB file (much quicker to send).

**Reply by jacobbieker** [reply]
Just data3 sounds good for now! Is there any important information missing from it? Like quality masks for the radar returns? We can always try adding that extra data later too

**Reply by ValterFallenius** [reply]
The quality masks are already applied on data3. Most important to know are gain and offset, 0.4 and -30 respectively.

I didn't have permission on the server to publish, awaiting permission to be granted ^^

/Valter

**Reply by jacobbieker** [reply]
Okay, great! Then yeah, just dat3 sounds great!

**Reply by ValterFallenius** [reply]
Here it is:

Export url:  http://exporter.nsc.liu.se/ddb5a8272e764e36a7d9daa632604aa1
rsync://exporter.nsc.liu.se/ddb5a8272e764e36a7d9daa632604aa1

**Reply by jacobbieker** [reply]
Awesome! Thanks so much!

**Reply by CUITCHENSIYU** [reply]
> Here it is:
> 
> Export url: http://exporter.nsc.liu.se/ddb5a8272e764e36a7d9daa632604aa1 rsync://exporter.nsc.liu.se/ddb5a8272e764e36a7d9daa632604aa1

Hello, the link seems to be broken.
